storage/time-series/influxdb/retriever/app/express/routes/v1/Devices.js

```diff
@@ -0,0 +1,80 @@
+const { Logger } = require('@dojot/microservice-sdk');
+const HttpStatus = require('http-status-codes');
+const util = require('util');
+const createError = require('http-errors');
+const { parseDateTimeToUnixNs } = require('../../../Utils');
+
+const logger = new Logger('influxdb-retriever:express/routes/v1/Device');
+
+/**
+ * Checks if "dateFrom" and/or "dateTo" are valid and if they are not returns an error
+ *
+ * @param {*} req
+ * @param {*} res
+ * @param {*} next
+ */
+const isDateValid = (req, res, next) => {
```

Comment:
This only serves to verify that the date is less than 1970 ... I think it is not necessary ...
Is it necessary to check if dateFrom is less than dateTo?